app/selenium_ui/confluence/pages/pages.py

```diff
@@ -83,6 +85,20 @@ def wait_for_comment_field(self):
     def click_edit(self):
         self.wait_until_clickable(PageLocators.edit_page_button).click()
 
+    def wait_for_resources_loaded(self):
+        sleep_time = 0.1
```

Comment:
It would be nice to have timeout user-friendly:
```
timeout = 5 
sleep_time = 0.1
attempts = timeout // sleep_time
```

Reply:
I think timeout should be measured with real time - not attempts with sleep. Please modify the function to use start_time and `while time.time() - start_time < timeout`